canvas_account_admin_tools/templates/canvas_account_admin_tools/dashboard_account.html

```diff
@@ -58,6 +58,13 @@ <h4 class="list-group-item-heading">Find Course Info</h4>
           </a>
         </div>
 
+        <div class="list-group col-md-4">
+          <a type="button" class="btn btn-primary list-group-item active" href="{% url 'cross_list_courses:index' %}" class="list-group-item active">
+              <h4 class="list-group-item-heading">Cross-list Courses</h4>
```

Comment:
_List_ is lowercase (Cross-list) in the ticket ACP and uppercase (Cross-List) in the mockups. @Vittorio2015 quick clarification on what it should be?